scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -2307,7 +2314,26 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
       val lhs = atPos(lhsStart, lhsEnd)(makeTupleTerm(lhses))
       if (lhs.is[Term.Repeated])
         syntaxError("repeated argument not allowed here", at = lhs.tokens.last.prev)
-      atPos(lhsStart, rhsEnd)(Term.ApplyInfix(lhs, op, targs, checkNoTripleDots(rhs)))
+
+      def infixExpression() =
+        atPos(lhsStart, rhsEnd)(Term.ApplyInfix(lhs, op, targs, checkNoTripleDots(rhs)))
+      op match {
+        case q: Quasi =>
```

Comment:
this case is not needed, `case _ =>` it the same

Reply:
Fixed